components/bill/invoice/invoice.templ

```diff
@@ -23,11 +23,14 @@ import (
 templ Invoice(env *gobl.Envelope, inv *bill.Invoice) {
 	// Footer needs to be at the top to show on all pages
 	<article class="invoice">
-		<header>
+		<header class={ templ.KV("qr", HasHeaderQR(env)), templ.KV("not-qr", !HasHeaderQR(env)) }>
```

Comment:
This is quite hard to read, looks like it should be in a function. For the class I'd also suggest affirmations with a bit more implication rather than negations, so `regular` as opposed to `not-qr`, or `with-qr` instead of just `qr`.